CHECLabPy/core/spectrum_fitter.py

```diff
@@ -199,11 +199,11 @@ def load_config(self, path):
             for c in self.coeff_names:
                 if 'initial' in d:
                     ini = c
-                    self.initial[ini] = d['initial'].pop(c, self.initial[ini])
```

Comment:
By using a yaml config file I stumbled upon the fact that yaml does not support scientific notation. The simple and easy solution is convert the values of limits and initial values to floats as we expect to formatted as valid floats in the config file.